sdk/appconfiguration/app-configuration/README.md

```diff
@@ -31,10 +35,8 @@ npm install @azure/ms-rest-nodeauth
 ##### Sample code
 
 ```typescript
-import * as coreHttp from "@azure/core-http";
-import * as coreArm from "@azure/core-arm";
 import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
```

Comment:
We recommend using `@azure/identity` with our new packages. I think the only one this library works with is the ManagedCredentials one from this package.

Reply:
Updated

Reply:
Updated to use connection string for now.